xbmc/cores/VideoPlayer/VideoRenderers/VideoShaders/YUV2RGBShader.cpp

```diff
@@ -94,8 +94,7 @@ void CalculateYUVMatrix(TransformMatrix &matrix
                         , unsigned int  flags
                         , ERenderFormat format
                         , float         black
-                        , float         contrast
-                        , bool          limited)
+                        , float         contrast)
```

Comment:
I think we should keep limited here. Looks like the signature in the header is wrong
@laurimyllari interesting that it compiles on Linux